package.json

```diff
@@ -39,17 +42,19 @@
     "react-dom": "18.3.1",
     "react-native": "0.76.6",
     "react-native-gesture-handler": "~2.20.2",
-    "react-native-reanimated": "~3.16.1",
+    "react-native-reanimated": "^3.16.7",
     "react-native-safe-area-context": "4.12.0",
-    "react-native-screens": "~4.4.0",
+    "react-native-screens": "^4.6.0",
```

Comment:
Running npx expo install --check will give the optimal versions for compatibility with expo. This should be "~4.4.0"